src/main/scripts/postToSlackCustom.groovy

```diff
@@ -21,35 +21,50 @@ final def props = airTool.getStepProperties()
 
 // properties
 final def webhook = props['webhook'];
-final def slackChannel = props['channel'];
+final def slackChannels = props['channels'];
 final def slackUsername = props['username'];
 final def emoji = props['emoji'];
 final def slackAttachment = props['attachment'];
 
 //Convert attachment input to be ArrayList for JSONBuilder
-def slurped = new JsonSlurper().parseText(slackAttachment)
-def attachmentJson = new JsonBuilder(slurped)
 
-attachmentJson.content[0].ts = "" + System.currentTimeMillis()/1000;
+slackChannels.eachLine {
+ 
+   def slackChannel = it
 
-// JSON message composition
-def json = new JsonBuilder();
-try {
-    json {
+   if (slackChannel == null || slackChannel.allWhitespace) {
+      return
+   }
+
+   if (!slackChannel.startsWith("@") && !slackChannel.startsWith("#")) {
+      return
```

Comment:
Quick println saying what you are skipping is helpful for debug. 

println "Warning::  The channel '${slackChannel}' is not formatted correctly. It must begin with an @ or # symbol."